Gemfile

```diff
@@ -28,12 +28,11 @@ gem 'strong_migrations', '~> 2.1.0'
 
 group :assets do
   gem 'coffee-rails', '~> 5.0'
-  gem 'non-stupid-digest-assets', '~> 1.0'
+  gem 'non-digest-assets', '~> 2.4.0'
```

Comment:
This gem was added a decade ago, mainly because of the need to use the `signup_v2.js` from outside of Rails (the signup form used to be included somewhere on 3scale.net website as an iframe).
https://github.com/3scale/system/pull/5690

Probably, it would be good to review - maybe we don't even need it anymore.

Reply:
Yes, unfortunately, this gem ended up being our only solution for making the background images work 🙃